packages/calling-component-bindings/src/errorBarSelector.ts

```diff
@@ -76,14 +76,15 @@ export const errorBarSelector: ErrorBarSelector = createSelector(
     if (deviceManager.deviceAccess?.audio === false && isSafari()) {
       activeErrorMessages.push({ type: 'callMicrophoneAccessDeniedSafari' });
     }
-    if (deviceManager.deviceAccess?.audio === false && !isSafari()) {
+    if (
```

Comment:
nit: this would be better moved to an `else` on the below `if (diagnostics?.media.latest.microphonePermissionDenied?.value === true && isMacOS()) {`
so it's clear that this condition is we don't push two microphone active errors